docs/content/guides/developer/sui-101/coin-mgt.mdx

```diff
@@ -1,6 +1,40 @@
 ---
 title: Coin Management
-draft: true
+description: Because Sui uses owned objects for transactions, you need to have plans to manage coins in your programmable transaction block development. 
```

Comment:
where is this **_description_** used?

Maybe we could refer to Coin here?
e.g Considering Coin are owned object, you need to explicitly manage them....